docs/user-guide/command-line-interface.md

```diff
@@ -69,7 +69,9 @@ Miscellaneous:
   --debug                    Output debugging information
   -h, --help                 Show help
   -v, --version              Outputs the version number
-  ```
+  --no-inline-config         Prevent comments from changing eslint rules -
```

Comment:
You are correct, but the default is true. I guess I figure people will want to use `—no-inline-config` on the command line as it is prettier, but I’ll fix the docs.